.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Please fill the `[ ]` with an `x` to make sure that you read and followed the checklist
+
+- [ ] I have checked that the proposed addition is not already in the awesome list
```

Comment:
`[ ] Proposed addition is related to Machine Learning on Source Code.`
Quite a dummy but it could help us a bit to avoid some of the PRs where people try to add `tensorflow`/`pytorch`, or some other stuff not related to MLonCode.
WDYT?

Reply:
I like this one a lot. :+1: